src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java

```diff
@@ -357,6 +357,7 @@ public void update() {
 					String clean = line.replaceAll("(\u00a7.)", "");
 					String[] cleanSplit = clean.split(" ");
 					hidden.cookieBuffRemaining = 0;
+					if (line.contains("Not")) break;
```

Comment:
Nooooooo

Reply:
Hi, this doesn't result in an incomplete parser (if thats what you are worried about), since the cookieBuffRemaining variable gets set back to 0 before this, which means that if you have "Not active" as cookie buff status, the remaining time gets set to 0 (which i think is correct). Yes regex would be nicer, but this isnt my codebase and the people that do have experience with this codebase didnt do anything about it, so i did this.